CODEOWNERS

```
@@ -688,6 +688,7 @@
 /scripts/print_toolchain_checksum.sh      @nrfconnect/ncs-ci
 /scripts/sdp/                             @nrfconnect/ncs-ll-ursus
 /scripts/twister/alt/zephyr/tests/drivers/mspi/api/testcase.yaml @nrfconnect/ncs-ll-ursus
+/scripts/generate_psa_key_attributes.py   @hakonfam
```

Use a team, not a person
```suggestion
/scripts/generate_psa_key_attributes.py   @nrfconnect/ncs-aurora
```

---

Fixed